core/kazoo_voicemail/src/kvm_message.erl

```diff
@@ -884,6 +886,7 @@ maybe_update_meta(Length, Action, Call, MediaId, BoxId) ->
         'delete' ->
             lager:debug("attachment was sent out via notification, set folder to delete"),
             Fun = [fun(JObj) ->
+                           'ok' = kvm_util:publish_voicemail_deleted(BoxId, JObj, 'delete_after_notify'),
```

Comment:
I think this is creating a bit noise, sending a delete email notification immediately after sending a new voice mail email.
If the user set the `delete_after_notify` option they already specified they don't want it in their in vmbox.

Reply:
@icehess this is happening because of the use case from @extremerotary.  They want to be able to bind to the amqp messages for deleted email, even if it happens in an automated way.

Reply:
@icehess is this acceptable?